### ISSUE TITLE
Problem with handleEvents option in makeServicePlugin method

### ISSUE THREAD

**Issue by s3ns3i** [feathersjs-ecosystem/feathers-vuex#472]
I have a problem, that is described there: https://github.com/feathersjs-ecosystem/feathers-vuex/issues/397
so I thought, that for now I would refresh other services by using _handleEvents_ property, but it seems to not appear in the state of the model (I don't know if it is supposed to), but also it's not working (I put console.log there, and it's not called).

### Steps to reproduce

This is what I did:

```
// store/services/roles.js
const servicePlugin = makeServicePlugin({
  Model: Role,
  service: feathersClient.service(servicePath),
  servicePath,
  handleEvents: {
    created: (item, { model, models }) => {
      console.log("roles: created event");
      return true;
    },
    ... // Same with the rest of handlers
  }
});
```

When it wasn't working, I also added this part:

```
// plugins/feathers.js
const {
  makeServicePlugin,
  makeAuthPlugin,
  BaseModel,
  models,
  FeathersVuex
} = feathersVuex(feathersClient, {
  serverAlias: "api"
  idField: "id" // Using sequelize
  debug: true,
  whitelist: ["$regex", "$options"],
  enableEvents: true,
  handleEvents: {
    created: () => true,
    updated: () => true,
    patched: () => true,
    removed: () => true
  }
});
```

Is there something I am doing wrong? It seems straightforward, but no luck so far.
If you will need more details, I will try to create a bigger example

### Expected behavior
Handlers are called when I create an object.

### Actual behavior
Handlers are not called.

### System configuration

**Module versions**

    "feathers-vuex": "^3.9.4",
    "@feathersjs/feathers": "^4.5.3",
    "@feathersjs/socketio-client": "^4.5.4",
    "@vue/composition-api": "^0.5.0",
    "feathers-hooks-common": "^5.0.3",
    "nuxt": "^2.12.2",
    "socket.io-client": "^2.3.0",

**NodeJS version**: v13.13.0

**Operating System**: Ubuntu 19.10

**Browser Version**: Mozilla 75.0

**Comment by J3m5** [comment]
Can you please check in your network panel if you are receiving the events ?
I don't see how it couldn't works...

**Comment by s3ns3i** [comment]
I don't see anything in my network panel, but when I create a role on one browser, it shows on the other one, so I assume, that sockets work.

I've also added a code from this section: https://vuex.feathersjs.com/model-classes.html#model-events - this one works (I can see logs in the console).

Can it be because I'm using nuxt? Maybe there is some additional configuration required?

**Comment by hamiltoes** [comment]
I think the offending code is in `make-service-plugin.ts`.

```ts
    const options = Object.assign({}, defaults, globalOptions, config)
    const {
      Model,
      service,
      namespace,
      nameStyle,
      instanceDefaults,
      setupInstance,
      preferUpdate
    } = options

    events.forEach(eventName => {
      if (options.handleEvents[eventName])
        options.handleEvents[eventName] = () => options.enableEvents || true
    })
```

The handlers passed in are being overridden. I think just negating the if statement would do the trick. (`if(!options.handleEvents[eventName])`)

**Comment by J3m5** [comment]
It might be caused by Nuxt, have you followed [the Nuxt chapter in the docs](https://vuex.feathersjs.com/nuxt.html#nuxt) ?

**Comment by s3ns3i** [comment]
Um, I've did like the @hamiltoes said, and voila! It works!

**Comment by J3m5** [comment]
Good catch @hamiltoes ! This bug must be addressed.

**Comment by hamiltoes** [comment]
I'm happy to put together a PR. I was thinking expected (and desired) behavior would be to fall back to any global option `handleEvent` method if one is defined.

**Comment by J3m5** [comment]
This bug was introduced [in this commit ](https://github.com/feathersjs-ecosystem/feathers-vuex/commit/154b71820738b6fe8cdd68a462f31cdbff0d083e#diff-cfab261362a6e936b0e2ca6390f4991cR43).
Before that, the default event handlers were added before the options where merged.

I think this is a good idea, thank you for your help, feel free to submit a PR !

**Comment by marshallswain** [comment]
The fix for this is available in `feathers-vuex@3.10.0`
